### DISCUSSION TITLE
유니티로 짚라인 구현 후, 제페토에서 활용 가능할까요?

### DISCUSSION THREAD

**Discussion by seo1seo1** [naverz/zepeto-studio-kor#920] [5. Q&A]
출발점 - 도착점 두 곳에 상호작용 오브젝트를 설치하고

상호작용 시 목표지점에 일직선으로 등속이동하게끔 구현하려합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 도착점의 pos으로 MoveToPosition해서 구현 가능합니다.
아래 가이드를 참고 부탁 드립니다.
https://studio.zepeto.me/guides/zepetocharacter
감사합니다.
